en/ios/user-interface.mdown

```diff
@@ -484,8 +484,56 @@ The easiest way to understand this class is with an example. This subclass of `P
 @end
 ```
 ```swift
-// no Swift example
-// Want to contribute to this doc? https://github.com/ParsePlatform/Docs
+class SimpleTableViewController : PFQueryTableViewController {
+    
+    override init(style: UITableViewStyle, className: String?) {
+        super.init(style: style, className: className)
+        self.parseClassName = "Todo"
+        self.pullToRefreshEnabled = true
+        self.paginationEnabled = true
+        self.objectsPerPage = 25
```

Comment:
For all of these properties - you can simply use them directly without using `self.` - it will use the setter for that property, since Swift is smart. Also - since we have IDEs (woohoo!) it will highlight them as they are instance variables (usually green color), so it's absolutely fine to simply use them directly.